README.md

```diff
@@ -53,54 +42,6 @@ Eclipse is packaged with the [Android Developer Tools bundle](http://developer.a
 
 ## Troubleshooting
 ### I see a grey map
-
-Follow the instructions under "Add your API key" and try again.
-
-### I see the Android starter screen
-  1. Choose Window>Close All Perspectives
-  2. Choose Window>Open Perspective>Other...
-  3. Choose Java
-
-# IntelliJ
-## Download IntelliJ
-http://www.jetbrains.com/idea/download/
-Choose the "Community Edition"
-
-## Import the project
-  1. In the start up screen, choose "Open project"
-  2. Select the checked out project, press OK
-
-## Add your API key
-  1. Navigate to the AndroidManifest.xml file
-  2. Replace "your_api_key" with an API key generated using [these instructions](https://developers.google.com/maps/documentation/android/start#the_google_maps_api_key)
-
-## Run the app
-  1. Ensure your phone is in plugged in, developer mode enabled and screen unlocked
-  2. Choose Run>Edit Configurations...
-  3. Press the + button
-  4. Click "Android Application"
-  5. Edit the name
-  6. Choose "hellomap" as the Module
-  7. Choose "USB Device" as your Target Device
-  8. Press OK
-  9. Press the green play button next to the name you specified. This will build the app, install it on the device and run it.
-
-## Troubleshooting
-### "Please specify Android SDK"
-  1. Click "New..."
-  2. Choose "Android SDK"
-  3. Navigate to the Android SDK's "sdk" directory, press "Choose"
-
-###  "No Java SDK of appropriate version found"
-  1. Click "New..."
-  2. Choose "JDK"
-  3. Navigate to the JDK's home directory. For OS X, this is at `/Library/Java/JavaVirtualMachines/jdk.../Contents/Home`
-  4. Press "Choose"
-  5. If you are prompted with "Set up created SDK on project?", press No.
-  6. Follow the instructions above
-  7. If you are prompted with "Set up created SDK on project?", press Yes.
-
-### I see a grey map
 Follow the instructions under "Add your API key" and try again.
```

Comment:
Perhaps add a line here suggesting that the developer should also keep an eye on the logcat output, as the API will often provide extra information there, e.g. if the cert/key aren't set up correctly.